flow/renderer/pyglet_renderer.py

```diff
@@ -55,8 +75,37 @@ def __init__(self, network, mode,
                  path=HOME+"/flow_rendering",
                  sight_radius=50,
                  show_radius=False,
-                 pxpm=2):
-        """Instantiate a pyglet renderer class."""
+                 pxpm=2,
+                 alpha=1.0):
+        """Initialize Pyglet Renderer.
+
+        Parameters
+        ----------
+        network : list
+            A list of road network polygons
+        mode : str or bool
+
+            * False: no rendering
+            * True: delegate rendering to sumo-gui for back-compatibility
+            * "gray": static grayscale rendering, which is good for training
+            * "dgray": dynamic grayscale rendering
+            * "rgb": static RGB rendering
+            * "drgb": dynamic RGB rendering, which is good for visualization
+
+        save_render : bool
+            Specify whether to save rendering data to disk
+        path : str
+            Specify where to store the rendering data
+        sight_radius : int
+            Set the radius of observation for RL vehicles (meter)
+        show_radius : bool
+            Specify whether to render the radius of RL observation
+        pxpm : int
+            Specify rendering resolution (pixel / meter)
+        alpha: int
+            Specify opacity of the alpha channel.
```

Comment:
what is the alpha channel?

Reply:
Alpha channel controls transparency of each visual element. For more info, check https://en.wikipedia.org/wiki/Alpha_compositing